src/components/__tests__/RegistryAdmin.test.tsx

```diff
@@ -88,40 +97,60 @@ describe('RegistryAdmin, with access token', () => {
 
     expect(libraryAccordions.length).toEqual(15);
   });
+
+  afterEach(() => {
+    jest.clearAllMocks();
```

Comment:
I think we can enable this globally in the jest.config.js file by adding  `clearMocks: true` so that we don't have to add it to each test.